<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serverTablo</class>
 <widget class="QMainWindow" name="serverTablo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>serverTablo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="sb_seconds">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>55</pointsize>
     </font>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <number>-1</number>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sb_minutes">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>55</pointsize>
     </font>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <number>-1</number>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sb_hours">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>55</pointsize>
     </font>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <number>-1</number>
    </property>
    <property name="maximum">
     <number>23</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_resetTime">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сброс</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_startpause">
    <property name="geometry">
     <rect>
      <x>388</x>
      <y>230</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>21</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>65</pointsize>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>21</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>65</pointsize>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbScoreTeamOne">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbScoreTeamTwo">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditHistory">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>341</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNameTeamOne">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Клуб 1</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNameTeamTwo">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Клуб 2</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>128</y>
      <width>345</width>
      <height>86</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Время матча</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResetAllData">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сбросить всё</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGoalTeamOne">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>424</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Забить гол</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditGoalTeamOne">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelGoalTeamOne">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>330</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Автор гола</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGoalTeamTwo">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>424</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Забить гол</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditGoalTeamTwo">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>370</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelGoalTeamTwo">
    <property name="geometry">
     <rect>
      <x>566</x>
      <y>330</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Автор гола</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>sb_seconds</zorder>
   <zorder>sb_minutes</zorder>
   <zorder>sb_hours</zorder>
   <zorder>btn_resetTime</zorder>
   <zorder>btn_startpause</zorder>
   <zorder>label1</zorder>
   <zorder>label2</zorder>
   <zorder>sbScoreTeamOne</zorder>
   <zorder>sbScoreTeamTwo</zorder>
   <zorder>textEditHistory</zorder>
   <zorder>lineEditNameTeamOne</zorder>
   <zorder>lineEditNameTeamTwo</zorder>
   <zorder>label</zorder>
   <zorder>btnResetAllData</zorder>
   <zorder>btnGoalTeamOne</zorder>
   <zorder>lineEditGoalTeamOne</zorder>
   <zorder>labelGoalTeamOne</zorder>
   <zorder>btnGoalTeamTwo</zorder>
   <zorder>lineEditGoalTeamTwo</zorder>
   <zorder>labelGoalTeamTwo</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
